src/firefox/remote.js

```diff
@@ -174,8 +174,8 @@ export class RemoteFirefox {
     const addon = await this.getInstalledAddon(addonId);
     await this.checkForAddonReloading(addon);
     await this.addonRequest(addon, 'reload');
-    log.info(
-      `${(new Date()).toTimeString()}: Reloaded extension: ${addon.id}`);
+    process.stdout.write(`${'\r' +
+      'Last extension reload: '} ${(new Date()).toTimeString()}`);
```

Comment:
I take it you were trying to defeat the line length rule? I think it would be easier to read if you concatenated string templates instead:
````javascript
process.stdout.write(
  `${'\rLast extension reload: '} ` +
  `${(new Date()).toTimeString()}`);

````